libcontainer/cgroups/fs/stats_util_test.go

```diff
@@ -121,3 +121,47 @@ func expectMemoryDataEquals(t *testing.T, expected, actual cgroups.MemoryData) {
 		t.Fail()
 	}
 }
+
+func expectCPUUsageEqual(t *testing.T, expected, actual cgroups.CpuUsage) {
+	if expected.TotalUsage != actual.TotalUsage {
+		logrus.Printf("Expected CPU total usage %d but found %d\n",
+			expected.TotalUsage, actual.TotalUsage)
+		t.Fail()
```

Comment:
Not sure why logrus is used here, but `testing.T` has its own logging functions, and log+fail functions (such as https://golang.org/pkg/testing/#T.Errorf)

IOW the above might be a one liner:
```
t.Errorf("Expected CPU total usage %d but found %d\n",
￼			expected.TotalUsage, actual.TotalUsage)
```

Reply:
Logging in other places in the file suggested me to use logrus but you're absolutely right so I've removed logrus logs in the whole file and I've used `t.Errorf()`